runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/MonitoringUtil.java

```diff
@@ -195,9 +196,9 @@ public static String getJobMonitoringPageURL(String projectName, String regionId
       // that has the project name replaced with project id.
       return String.format(
           "https://console.cloud.google.com/dataflow/jobsDetail/locations/%s/jobs/%s?project=%s",
-          URLEncoder.encode(regionId, "UTF-8"),
-          URLEncoder.encode(jobId, "UTF-8"),
-          URLEncoder.encode(projectName, "UTF-8"));
+          URLEncoder.encode(regionId, StandardCharsets.UTF_8.name()),
```

Comment:
Is it better than `StandardCharsets.UTF_8.toString()` ?

Reply:
According to the Javadoc, name() "Returns this charset's canonical name. ". So I think it's fine, and also it's used elsewhere in the codebase.

Reply:
Fair enough. It sounds good, thanks !